src/qibolab/instruments/emulator/backends/generic.py

```diff
@@ -0,0 +1,315 @@
+    print("-" * 21)
+
+    print("Two-body drift terms:")
+    print("-" * 21)
+    if len(model_config["drift"]["two_body"]) == 0:
+        print("None")
+    for op_instruction in model_config["drift"]["two_body"]:
+        op_instruction = op_from_instruction(
+            op_instruction,
+            latex_op_dict,
+            latex_op_connectors_dict,
+            multiply_coeff=False,
+        )
+        inst = format_instruction(op_instruction)
+        display(Latex(inst[0]))
+        display(Latex(inst[1]))
+    print("-" * 21)
+
+    print("One-body drive terms:")
+    print("-" * 21)
+    for drive_instructions in model_config["drive"].values():
+        for op_instruction in drive_instructions:
+            op_instruction = op_from_instruction(
+                op_instruction,
+                latex_op_dict,
+                latex_op_connectors_dict,
+                multiply_coeff=False,
+            )
+            inst = format_instruction(op_instruction)
+            display(Latex(inst[0]))
+            display(Latex(inst[1]))
+    print("-" * 21)
+
+    print("Dissipative terms:")
+    print("-" * 21)
+    for key in model_config["dissipation"].keys():
+        print(">>", key, "Linblad operators:")
+        if len(model_config["dissipation"][key]) == 0:
+            print("None")
+        for op_instruction in model_config["dissipation"][key]:
+            op_instruction = op_from_instruction(
+                op_instruction,
+                latex_op_dict,
+                latex_op_connectors_dict,
+                multiply_coeff=False,
+            )
+            inst = format_instruction(op_instruction, is_dissipation=True)
+            display(Latex(inst[0]))
+            display(Latex(inst[1]))
+    print("-" * 21)
+
+
+def plot_fidelity_history(
+    time_list: list,
+    fid_list_all: list,
+    labels: list,
+    time_in_dt: bool = False,
+) -> Figure:
+    """Plots the fidelity history.
+
+    Args:
+        time_list (list): Specifies the list of times used in simulation.
+        fid_list_all (list): List of fidelity histories.
+        labels (list): List of labels. Displayed in the plot legend.
+        time_in_dt (bool): Specify the units of the x-axis in the plots to be in dt (inverse sampling rate) if True and in ns if False. Defaults to False.
+
+    Returns:
+        Figure: Figure of input fidelity histories with labels.
+    """
+
+    fig = plt.figure()
+    for result_ind in range(len(labels)):
+        plt.plot(
+            time_list,
+            fid_list_all[result_ind],
+            label=labels[result_ind],
+        )
+    plt.legend(loc="upper left")
+    plt.ylabel("Overlap with basis state")
+    if time_in_dt:
+        plt.xlabel("Time / dt")
+    else:
+        plt.xlabel("Time / ns")
+    for result_ind in range(len(labels)):
+        print(
+            labels[result_ind],
+            fid_list_all[result_ind][0],
+            fid_list_all[result_ind][-1],
+        )
+
+    return fig
```

Comment:
Same remarks of the above.

Reply:
Fixed in 0d5129a784f1ed495f971e27fa778fe4aab854c6